pkgs/tools/virtualization/xen-guest-agent/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitLab,
+  rustPlatform,
+  llvmPackages,
+  xen-light,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "xen-guest-agent";
+  version = "0.3.0-unstable-2023-12-08";
+
+  src = fetchFromGitLab {
+    owner = "xen-project";
+    repo = pname;
+    rev = "e6e165e74ce161704af7c96966a26f62a93bdbed";
+    hash = "sha256-MUcYtrLJdTkcgBtjHv1wcOJQxK0xNb9SItEo6yBWJMU=";
+  };
+
+  cargoHash = "sha256-pov7+mmlpu/PnbtY40yWh9KskbcP3SDpVailarARY6E=";
+
+  LIBCLANG_PATH="${llvmPackages.libclang.lib}/lib";
```

Comment:
Env variables have, like the other things, usually a space before and after the equals sign and should ideally be under the env attr.

Reply:
fixed, thank you!